### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by ncnynl**
Change entry script call

### CODE REVIEW

File: micro-ROS-static-library-builder/Dockerfile [hunk 1]
```diff
@@ -40,4 +40,5 @@ RUN git clone -b iron https://github.com/micro-ROS/micro-ros-build.git src/micro
 
 COPY ./entrypoint.sh /entrypoint.sh
 
-ENTRYPOINT ["/entrypoint.sh"]
+#ENTRYPOINT ["/entrypoint.sh"]
```
Comment (pablogs9) [inline micro-ROS-static-library-builder/Dockerfile]:
```suggestion
```

Not required

**Review by pablogs9 — CHANGES_REQUESTED** [review]

**Review by ncnynl — COMMENTED** [review]
Remove redundant code

**Comment by pablogs9** [comment]
Could you explain the rationale of this PR a bit?

**Review by ncnynl — COMMENTED** [review]
ENTRYPOINT ["/entrypoint.sh"] it will have permission issues, causing it to not work properly when running docker container.  so change into ENTRYPOINT ["/bin/sh", "/entrypoint.sh"] can fixed it

**Comment by pablogs9** [comment]
@mergify backport humble rolling

**Pull request merged**